flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -176,5 +176,23 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table_2.10</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-scala_2.10</artifactId>
```

Comment:
You're correct.

The problem is that in order to actually use the connector in a meaningful way users do need to depend on the packages, as `ExecutionEnvironment` and `StreamExecutionEnviornment` are exposed in the APIs:

```
$ javap flink-libraries/flink-table/target/classes/org/apache/flink/table/api/TableEnvironment.class
```

```
  public static org.apache.flink.table.api.scala.BatchTableEnvironment getTableEnvironment(org.apache.flink.api.scala.ExecutionEnvironment);
  public static org.apache.flink.table.api.java.BatchTableEnvironment getTableEnvironment(org.apache.flink.api.java.ExecutionEnvironment, org.apache.flink.table.api.TableConfig);
  public static org.apache.flink.table.api.java.BatchTableEnvironment getTableEnvironment(org.apache.flink.api.java.ExecutionEnvironment);
```

This is captured by the integration test in the patch. However, I do think these two dependency should be `test` instead of `provided`. At the very least the dependency does not get leaked to the users.

@PangZhi can you please make the changes?